### ISSUE TITLE
screen reader Unlabled link at the top of discussion threads

### ISSUE THREAD

**Issue by smoke-j** [flarum/core#1835]
When in a discussion thread there is a link at the top of the page which only reads as "link" 
For example: on the page located at
https://discuss.flarum.org/d/20733-screen-reader-accessibility/2
There is a link at the top of the page inside the banner landmark that points to the home page of 
https://discuss.flarum.org/
The link text just reads as "link"
The unlabeled link takes you to the same place the image link labeled as "Flarum Community" in the banner takes you.
I have tested this with the two most common windows screen readers, NVDA and Jaws.
With both NVDA and Jaws in both chrome and Firefox the unlabeled link is read as just "link".
With jaws the image link is not read at all, but that may be due to my jaws configuration.

**Comment by franzliedke** [comment]
@smoke-j Thanks for the report!

Are you talking about the link in the navigation bar right before "Home", "Donate" and "Docs"? That would be our logo, which is a `<h1>` that contains a link which wraps an image with a proper `alt` attribute. I wonder why the screenreader does not read this attribute?

**Comment by smoke-j** [comment]
It seems to be doubling the link. There is the normal labled graphic which reads as.
"heading  level 1visited  linkgraphicFlarum Community"
but right above that is an empty link which just reads as
"banner landmark visited  link"
they both go to the same place, but the one at the very top of the page is not labeled, the graphic link is labled perfect. for some reason it is just showing up as two links. the non labeled one then the properly labeled graphical link.

**Comment by franzliedke** [comment]
Ah right, that's our own "Back" button. It does have a `title` attribute of "Back to Discussion List" - I wonder why that isn't read out?

**Comment by franzliedke** [comment]
Oh, [this article explains why](https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong) - we should add explicit descriptive text, the `title` attribute is mostly ignored by screenreaders.

Possible ways to do this are [explained here](https://webaim.org/techniques/css/invisiblecontent/) - though maybe an `aria-label` will help as well.

**Comment by franzliedke** [comment]
I just pushed a fix that could potentially fix this. Once it is live on our demo site https://nightly.flarum.site (probably early Friday morning), please report back here whether anything improved. :)

**Comment by franzliedke** [comment]
@smoke-j The change is live on https://nightly.flarum.site/d/1-new-discussion-hey. Can I ask you to test it with a screenreader?

**Comment by smoke-j** [comment]
It seems to be perfect. there is just a heading at the top of the nightly page now.

I know it probably doesn't seem like a big deal, but empty links are a big issue nowadays with a lot of modern sites. A lot of people get touchy about it.

**Comment by franzliedke** [comment]
No "Back to discussion list" link?

(If the link stops working, just register a new account with a non-existent email address starting with "admin@" and create a new discussion.)

**Comment by franzliedke** [comment]
Feedback was positive, so I consider this fixed. Thanks for bringing it up, @smoke-j !
